models/prices/prices_unsupported_chains_non_native_tokens.sql

```diff
@@ -0,0 +1,25 @@
+{{ config(
+        schema='prices_unsupported_chains_non_native',
+        alias = 'tokens',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static']
+        )
+}}
+SELECT 
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('btcs-ordinals', 'brc-20', 'BTCS', 'edc052335f914ee47a758cff988494fbb569d820e66ac8581008e44b26dcdb43i0', null),
```

Comment:
If the `decimals` value is unknown, you can cast null to a certain type.
```suggestion
    ('btcs-ordinals', 'brc-20', 'BTCS', 'edc052335f914ee47a758cff988494fbb569d820e66ac8581008e44b26dcdb43i0', CAST(NULL AS INT256)),
```

Reply:
GM,
Inspired by prices_native_tokens.sql, I did the decimals cast at line 14 with ", CAST(decimals as int) as decimals".